apps/agent/app/src/components/Transactions.js

```diff
@@ -0,0 +1,479 @@
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.surfaceContent};
+            `}
+          >
+            {format(date, 'MM/dd/yy')}
+          </time>
+        )
+        const badgeDiv = onlyOne ? (
+          <div
+            css={`
+              ${layoutName === 'medium' &&
+                `
+                  display: inline-flex;
+                  max-width: 150px;
+                `}
+              ${layoutName === 'large' &&
+                `
+                  display: inline-flex;
+                  max-width: 'unset';
+                `}
+            `}
+          >
+            <LocalIdentityBadge
+              connectedAccount={addressesEqual(entity, connectedAccount)}
+              networkType={network && network.type}
+              entity={entity}
+            />
+          </div>
+        ) : (
+          <LocalIdentityBadge
+            badgeOnly={typeof targetContract !== 'string'}
+            entity={
+              (!tokenTransfers.length && targetContract) || 'Multiple accounts'
+            }
+            networkType={
+              typeof targetContract === 'string' && network ? network.type : ''
+            }
+          />
+        )
+        const typeDiv = (
+          <div
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.surfaceContent};
+            `}
+          >
+            {TRANSACTION_TYPES_LABELS[type]}
+          </div>
+        )
+        const referenceDiv = (
+          <div
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.surfaceContent};
+            `}
+          >
+            {reference}
+          </div>
+        )
+        const amountDiv = (() => {
+          if (!onlyOne) {
+            const uniqueTokens = Object.keys(
+              tokenTransfers.reduce((p, { token }) => {
+                p[token] = 1
+                return p
+              }, {})
+            ).length
+            return (
+              <div
+                css={`
+                  ${textStyle('body2')};
+                  color: ${theme.surfaceContent};
+                `}
+              >
+                {uniqueTokens} token
+                {uniqueTokens > 1 || uniqueTokens === 0 ? 's' : ''}
+              </div>
+            )
+          }
+
+          const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
+          const formattedAmount = formatTokenAmount(
+            amount,
+            isIncoming,
+            decimals,
+            true,
+            { rounding: 5 }
+          )
+          const amountColor = isIncoming
+            ? theme.positive
+            : formattedAmount.includes('-')
+            ? theme.negative
+            : theme.surfaceContent
+          return (
+            <span
+              css={`
+                ${textStyle('body2')};
+                color: ${amountColor};
+              `}
+            >
+              {formattedAmount} {symbol}
```

Comment:
We should add a non-breaking space here as well :slightly_smiling_face: 

```suggestion
              {formattedAmount} {symbol}
```